python/lsst/obs/cfht/cfhtIsrTask.py

```diff
@@ -67,10 +68,11 @@ def run(self, ccdExposure, bias=None, linearizer=None, dark=None, flat=None, def
         metadata = floatExposure.getMetadata()
 
         # Detect saturation
-        # Saturation values recorded in the fits hader is not reliable, try to estimate it from
-        # the pixel vales
-        # Find the peak location in the high end part the pixel values' histogram and set the saturation
-        # level at safe * (peak location) where safe is a configurable parameter (typically 0.95)
+        # Saturation values recorded in the fits hader is not reliable, try to
+        # estimate it from the pixel values.
+        # Find the peak location in the high end part the pixel values'
+        # histogram and set the saturation level at safe * (peak location)
+        # where safe is a configurable parameter (typically 0.95)
```

Comment:
I thought only doc strings were required to be short so I'm a bit surprised by this change. Not that it's a problem, but I hope we won't be expected to do this for all our code.

Reply:
We have to include comments because PEP8 includes comments and docstrings in the definition and pycodestyle implements the PEP8 rule. When we enable max docstring rule in flake8 the rule can't tell the difference. The developer guide does specify comments and docstrings.